background.js

```diff
@@ -1,3 +1,19 @@
 chrome.runtime.onInstalled.addListener(() => {
-
+    
 });
+
+//lists items that should be added to list when popup opens
+let itemsToAdd = [];
+
+//listener for highlight and popup open
+chrome.runtime.onMessage.addListener(function(request, sender, sendResponse){
+    if (request.text == "popup opened") {
```

Comment:
`===` is definitely the way to go here :)

While `==` works fine, it's clearer to the reader that you're doing a strict comparison if you use the `===` operator.

Further reading: https://stackoverflow.com/questions/359494/which-equals-operator-vs-should-be-used-in-javascript-comparisons?noredirect=1&lq=1